### PR TITLE
Create staff via the API.

### PR DESCRIPTION
**Pull request by trauty-is-me**

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -38,5 +38,7 @@ tags:
       description: These are the times that have been approved and will be used in payroll.
     - name: Stats
       description: Location statistics to assist with accurate scheduling.
+    - name: Background Job Processor
```
Comment (jaskinn) [inline spec/openapi.yaml]: I'm not completely happy with the "Background Job Processor". What about "Bulk Importer"? I know this then potentially limits us to use it only for importing and we have already been asked about syncing staff...

I think it is the word "background", I don't feel like giving out that facet of information for some reason, it makes me uneasy.

Maybe "Mass Processor", or "Batch Processor", or "Bulk Processor"? Open to ideas.

Comment (trauty-is-me) [reply spec/openapi.yaml]: I am happy with any of the three you suggested.

Comment (trauty-is-me) [reply spec/openapi.yaml]: Batch is my preference

**Comment by jaskinn** [comment]
It looks like there is a linter error on travisCI

`
 Lint error:

/paths/~1api~1status~1{batch_id}/get

spec/paths/api@status@{batch_id}.yaml:

  1) #/get Templated parameter batch_id not found
`

**Pull request merged**